docs/planning-experiments/entry_points.md

```diff
@@ -0,0 +1,26 @@
+# Entry Points
```

Comment:
Wonder if we can fold this into the 'setting up the sample size calculator' page -- I feel like browsing through the pages content is a bit spread out across pages and it's not really clear where to start. Of course I can read through all of it but a "quickstart" flow would be useful I think, now it feels like the important data is spread across multiple pages

Reply:
I added a quickstart to the index page + I folded entry points into the "Setting up the Sample Size Calculator" page.